build-info-extractor-maven3/src/test/java/org/jfrog/build/extractor/maven/resolver/ArtifactoryResolutionTest.java

```diff
@@ -45,7 +45,7 @@ public class ArtifactoryResolutionTest {
     public void setup() {
         proxySelector = new ProxySelector(HTTP_PROXY_URL, HTTP_PROXY_PORT, HTTP_PROXY_USERNAME, HTTP_PROXY_PASSWORD, HTTPS_PROXY_URL, HTTPS_PROXY_PORT, HTTPS_PROXY_USERNAME, HTTPS_PROXY_PASSWORD, NO_PROXY_PATTERN);
         artifactoryResolution = new ArtifactoryResolution(HTTPS_RELEASE_URL, SNAPSHOT_URL, USERNAME, PASSWORD, proxySelector, new NullPlexusLog());
-        snapshotRepository = artifactoryResolution.createSnapshotRepository();
+        snapshotRepository = artifactoryResolution.createSnapshotRepository(true, null);
```

Comment:
Perhaps a utility function to create a default snapshot repository would be useful here